larq/models.py

```diff
@@ -41,6 +41,20 @@ def _count_fp_weights(layer):
     return layer.count_params()
 
 
+def _bit_to_MB(bit_value):
+    bittobyte = 1.0 / 8.0
+    mbconverter = 1024 ** 2
+    return bit_value * bittobyte / mbconverter
+
```

Comment:
I think if we choose the name of variables self-explanatory, it will be a lot easier to read the code.
```suggestion
bit_to_byte_ratio = 1.0 / 8.0
byte_to_mega_bytes_ratio = 1.0 / (1024**2)
return bit_value * bit_to_byte_ratio * byte_to_mega_bytes_ratio
```